docs/network-flow-visibility.md

```diff
@@ -230,6 +258,8 @@ Collector in production.
 
 #### ClickHouse Configuration
 
+##### Service Customization
```

Comment:
A general comment on all of the original configuration instructions: If we keep all these original configuration instructions for users who don't to use Helm, can we write it explicitly in the doc, with something like: for all of the configurations, option 1 - with Helm, edit values.yaml, option 2 - without Helm, edit flow-visibility.yml directly.

Otherwise it seems a bit confusing to me, if we recommend using Helm to template yaml, and we also recommend making a series of configurations directly on flow-visibility.yml in the meanwhile.

Reply:
I make an update on the document, hope it clearer now.